Contributing.md

```diff
@@ -1,37 +1,79 @@
-# Contributing to the Redistricting project
-
-Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
-
-## PROJECT DESCRIPTION
-Please refer to [README.md](README.md)
-
-## PROJECT WORKFLOW
-Please refer to the [Project Kanban Board](https://github.com/MetroCS/redistricting/projects/1)
-
-## PROJECT DISCUSSIONS
-This project uses [Discussions](https://github.com/MetroCS/redistricting/discussions) for project-related announcements, questions and answers, ideas, and engaging in all discussions not related to a specific project [Issue](https://github.com/MetroCS/redistricting/issues) or [Pull Request](https://github.com/MetroCS/redistricting/pulls).
-
-## TECHNOLOGIES
-* Programming Language - Java
-* Issue Tracking - GitHub
-* Version Control - Git
-* Build Process - Apache Ant 
-
-## TOOLS
-### Git
-* [Git](https://git-scm.com)  
-* [Git Clients](https://git-scm.com/downloads)
-
-### GitHub
-* [GitHub](https://github.com/)
-* [GitHub Tutorial](https://guides.github.com/activities/hello-world/) - "Hello World" starter guide for those new to GitHub
-
-### Apache Ant 
-* [Apache Ant](https://ant.apache.org/bindownload.cgi)
-
-#### Using Ant at the command line 
-```
-   $ cd <directory containing build.xml>
-   $ ant all 
-   $ cd build
-```
+# Contributing to the Redistricting project
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## PROJECT DESCRIPTION
+Please refer to [README.md](README.md)
+
+## PROJECT WORKFLOW
+Please refer to the [Project Kanban Board](https://github.com/MetroCS/redistricting/projects/1)
+
+## PROJECT DISCUSSIONS
+This project uses [Discussions](https://github.com/MetroCS/redistricting/discussions) for project-related announcements, questions and answers, ideas, and engaging in all discussions not related to a specific project [Issue](https://github.com/MetroCS/redistricting/issues) or [Pull Request](https://github.com/MetroCS/redistricting/pulls).
+
+## TECHNOLOGIES
+* Programming Language - Java
+* Issue Tracking - GitHub
+* Version Control - Git
+* Build Process - Apache Ant 
+
+## TOOLS
+
+### Git
+* [Git](https://git-scm.com)  
+* [Git Clients](https://git-scm.com/downloads)
+
+### GitHub
+* [GitHub](https://github.com/)
+* [GitHub Tutorial](https://guides.github.com/activities/hello-world/) - "Hello World" starter guide for those new to GitHub
+
+### Apache Ant: Version 1.10.7
+* Note: Version 1.10.7 does not allow the user to execute
+```
+   $ ant test
+
+```
+*  However, Version 1.10.12 does allow the test to run.
```

Comment:
Given this note and the fact that Apache Ant is not part of the repository, why not simply specify that Version 1.10.12 is the version to be used?

That also obviates the need for lines showing `$ ant test`.